concepts/exceptions/about.md

```diff
@@ -0,0 +1,140 @@
+# About
+
+The Java programming language uses [_exceptions_][exceptions] to handle errors and other exceptional events.
+
+## What is an exception
+
+An exception is an event that occurs during the execution of a program that disrupts the normal flow of instructions.
+
+Java distinguishes three types of exceptions:
+
+1. Checked exceptions
+2. Unchecked exceptions
+3. Errors
+
+### Checked exceptions
+
+_Checked exceptions_ are exceptional conditions that an application should anticipate and recover from.
+An example of a checked exception is the `FileNotFoundException` which occurs when a method is trying to read a file that does not exist.
+
+This type of exception is checked at compile-time: methods that throw checked exceptions should specify this in their method signature, and code calling a method that might throw a checked exception is required to handle it or the code will not compile.
+
+All exceptions in Java that do not inherit from `RuntimeException` or `Error` are considered checked exceptions.
+
+### Unchecked exceptions
+
+_Unchecked exceptions_ are exceptional conditions that an application usually cannot anticipate or recover from.
+An example of an unchecked exception is the `NullPointerException` which occurs when a method that is expecting a non-null value but receives `null`.
+
+This type of exception is not checked at compile-time: methods that throw unchecked exceptions are not required to specify this in their method signature, and code calling a method that might throw an unchecked exception is not required to handle it.
+
+All exceptions in Java that inherit from `RuntimeException` are considered unchecked exceptions.
+
+### Errors
+
+_Errors_ are exceptional conditions that are external to an application.
+An example of an error is the `OutOfMemoryError` which occurs when an application is trying to use more memory than is available on the system.
+
+Like unchecked exceptions, errors are not checked at compile-time. They are not usually thrown from application code.
+
+All exceptions in Java that inherit from `Error` are considered errors.
+
+## Throwing exceptions
+
+A method in Java can throw an exception by using the [`throw` statement][throw-statement].
```

Comment:
May want to explain what throwing an exception means. There are languages that don't have runtime exceptions or explicit throwing. Rust for example only has panic!, Go doesn't throw exceptions but _returns_ errors.